iis/iis.go

```diff
@@ -519,33 +540,36 @@ func applySiteBindings(siteName string, bindings []iisBinding) error {
 	properties := []string{"set", "site", siteName}
 
 	// Compare current bindings with desired bindings
-	// Remove any bindings that exist in both arrays
+	// Remove any bindings that exist in both arrays from currentBindings. This allows us to determine which of the currentBindings are no longer needed.
 	var exists bool
-	cIndex := 0
-	for _, currentBinding := range currentBindings {
+	for _, binding := range bindings {
 		exists = false
+		if binding.IPAddress == "" {
+			binding.IPAddress = "*"
+		}
```

Comment:
Is this also needed? https://github.com/Roblox/nomad-driver-iis/blob/ea2626acce5231706ec0fb730ea1538837cb23dd/iis/iis.go#L553-L555

Seems like this same check is in two places.

Reply:
Oop, you are totally right. I swapped the placement and forgot to remove the redundant one.

Reply:
While you are at it, a small nitpick: Remove extra line [`here`](https://github.com/Roblox/nomad-driver-iis/blob/ea2626acce5231706ec0fb730ea1538837cb23dd/iis/iis.go#L560)